.travis.yml

```diff
@@ -31,6 +31,7 @@ install:
   - nvm install $TRAVIS_NODE_VERSION
   - pip install -U pip setuptools wheel
   - pip install -r requirements/main.txt -r requirements/deploy.txt -r requirements/docs.txt -r requirements/lint.txt -r requirements/tests.txt
+  - pip install -r requirements/dev.txt
```

Comment:
I'm honestly not _exactly_ sure what changed that required this, it seems `pip-tools` was being installed some other way before, might relate to bumping travis dist to `bionic`

Reply:
also while tracking this down I noticed that in CI we are installing everything into the virtualenv travis automatically creates for us, but then reinstalling it into the `.state/env` we construct... but never use.